src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
Dunno, the meaning is a bit different. MemoryLimit is more like MemoryHigh in actual meaning. You'd set MemoryMax to a lower value than MemoryLimit, because MemoryMax does not invoke oom killer and results in gradual degradation. After reading the docs I think we might want to avoid the guesswork, and simply allow users to set both MemoryLimit and MemoryMax separately (i.e. what this patch does).